workflow/rules/diffexp.smk

```diff
@@ -57,21 +57,25 @@ rule sleuth_diffexp:
             "results/plots/mean-var/{model}.mean-variance-plot.pdf",
             caption="../report/plot-mean-var.rst",
             category="QC",
+            labels={"model": "{model}-mean-variance-plot"},
```

Comment:
This is not a good label. Rather have multiple keys: `{"model": "{model}", "plot": "mean-variance"}`, analogously below.

Reply:
Noted. Done